src/jimple-frontend/AST/jimple_declaration.cpp

```diff
@@ -0,0 +1,44 @@
+//
+// Created by rafaelsamenezes on 21/09/2021.
```

Comment:
This header is not our standard. Could you please revise it?

Reply:
Removed it